pages/getting-started/first-component.mdx

```diff
@@ -1,27 +1,28 @@
 # Build your own component
 
-Let's create a **dropdown component** to add to the dashboard that we [created earlier](/getting-started/first-embeddable).
+<Pill><span>Takes ~5 mins</span></Pill>
 
-<ImageGrid width="50%" images={["/img/dropdown.png"]} />
+Embeddable ships with a **full suite of components**, but you can build **anything you need** on top.
 
-<Steps>
+Simply **copy and paste** the code below to create a simple, reusable **select menu**.
```

Comment:
The high density of bolded text here causes the bold parts to no longer stand out, but the text to look a bit wobbly instead. Maybe reduce the amount of bolded text in this page.